packages/fxa-payments-server/src/components/NewUserEmailForm/index.tsx

```diff
@@ -131,7 +131,7 @@ export const NewUserEmailForm = ({
       {...{ onChange }}
     >
       <Localized
-        id="new-user-sign-in-link"
```

Comment:
`new-user-sign-in-link` can be removed from `packages/fxa-payments-server/src/components/NewUserEmailForm/en.ftl` as it is no longer being used anywhere.

Reply:
Same comment, we aren't removing this just yet.